src/Commands/StartCommand.php

```diff
@@ -30,6 +32,20 @@ protected function configure()
 
     protected function execute(InputInterface $input, OutputInterface $output)
     {
+        if ($input->getOption('daemonize')) {
+            $isChild = $this->daemonize();
+            if (!$isChild) {
+                if ($output->isVerbose()) {
+                    $output->writeln('Daemonized');
+                }
+                return null;
```

Comment:
As [this project supports Symfony 5](https://github.com/php-pm/php-pm/blob/2.1.1/composer.json#L7), the return value of a commands `execute` method must be an integer. While it is [not part of the methods signature](https://github.com/symfony/console/blob/v5.1.3/Command/Command.php#L162) in the abstract `Command` class, it is still [validated and an exception will be thrown](https://github.com/symfony/console/blob/v5.1.3/Command/Command.php#L260) for any non-integer return value. So I suggest, you change this to `return 0;` along with the other return statement below.